app/services/get_remote_tags.rb

```diff
@@ -7,14 +7,7 @@ def initialize(options)
 
   def process
     response = get_request(object_url)
-    build_tags(response.body)
+    @tags = JSON.parse(response.body)['data'].collect { |tag| { :tag => tag['tag'] } }
```

Comment:
What is the data type of `JSON.parse(response.body)['data']`? Is it already an array? Do you simply convert the key from `'tag'` to `:tag`?